napari/layers/points/_points_utils.py

```diff
@@ -253,7 +252,43 @@ def fix_data_points(
     return points, ndim
 
 
-def coerce_symbols(array: Iterable) -> np.ndarray:
+def symbol_conversion(symbol: Union[str, Symbol]) -> Symbol:
+    """
+    Convert a string or Symbol to a Symbol instance.
+    """
+    if isinstance(symbol, str):
+        symbol = SYMBOL_ALIAS.get(symbol, symbol)
+    return Symbol(symbol)
+
+
+def create_symbol_dict(symbols: Union[np.ndarray, list]) -> dict:
+    """
+    Create a dictionary that maps raw symbols (keys) to their Symbol
+    instance counterpart (values).
+    """
+    unique_symbols = (
+        set(symbols) if isinstance(symbols, (list, np.ndarray)) else {symbols}
+    )
```

Comment:
```suggestion
    unique_symbols = set(symbols)
```

Based on usage

Reply:
You're right, I forgot that other cases were managed in other ways in `coerce_symbols`